### DISCUSSION TITLE
Can't connect to my pre release 7 server

### DISCUSSION THREAD

**Discussion by GinoX27** [Pryaxis/TShock#1875] [Q&A] [answered]
As said in the title I can't connect to my server. Clients get stucked on 0% when receiving tiles.

The only thing I'm getting in the console when someone tries to connect are these errors 

![image](https://user-images.githubusercontent.com/17114240/82740393-9f0f2f00-9d48-11ea-89c6-33c152fa78d6.png)

**Comment by hakusaro** [comment]
@GinoX27 does it work with a different world? can you load the world in an older version of TShock and try with an older version of Terraria?

**Comment by GinoX27** [comment, marked as answer]
There's no fix for that in the end. Dedicated solutions are to avoid for now (Steam Multiplayer is still working properly) 'til Re-Logic fix this error.

Source: Confirmed on their official discord

Known problems:
-Stuck on tile receiving (0%)
-Map **_may_** corrupt if you try to use it on either Re-Logic's or TShock's Dedicated Server
